Description: fixing several simple warnings 
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/main.cc
+++ b/main.cc
@@ -47,9 +47,9 @@
 void nappi(unsigned char b,int x,int y);    //  Keyboard callback
 
 void updatetimer(void); //  Updates the currentTime, row and pattern
-void loadfile(char *name,void *dest); // Simply loads a file
+void loadfile(const char *name,void *dest); // Simply loads a file
 
-main(int argc,char *argv[])
+int main(int argc,char *argv[])
 {
     char datapath[]="/usr/share/neandertaler/";
     if (chdir(datapath) == 0) {
@@ -77,8 +77,8 @@
     MikMod_RegisterAllDrivers();
     MikMod_RegisterLoader(&load_xm);
     md_mixfreq=44100;
-    MikMod_Init("");
-    piisi=Player_Load("data/far_away.xm",12,0);
+    MikMod_Init((char *) "");
+    piisi=Player_Load((char *) "data/far_away.xm",12,0);
     Player_SetVolume(100);
     Player_Start(piisi);
 #endif
@@ -189,7 +189,7 @@
     currentTime=long(aeka);
 }
 
-void loadfile(char *name,void *dest)
+void loadfile(const char *name,void *dest)
 {
     FILE    *f;
 
--- a/neandertaler_mainloop.cc
+++ b/neandertaler_mainloop.cc
@@ -570,7 +570,7 @@
 extern unsigned char slider4_raw[64*64];
 extern unsigned char slider5_raw[64*64];
 
-char *letters = "abcdefghijklmnopqrstuvxyz";
+const char *letters = "abcdefghijklmnopqrstuvxyz";
 int font_num_chars = 25;
 int font_pos[25*2] = {6,22,25,39,41,55,57,72,74,88,89,102,104,120,122,138,141,152,153,165,166,183,186,197,200,222,224,241,243,259,259,276,278,293,298,309,311,325,329,338,341,356,360,374,375,392,394,410,413,427};
 
@@ -607,7 +607,7 @@
     glTexImage2D (GL_TEXTURE_2D, 0, GL_RGBA, width, height, 0, GL_RGBA, GL_UNSIGNED_BYTE, memseg);
 }
 
-void putstring(char *str, unsigned char *dest, int posx, int posy, int destwidth, int destheight) {
+void putstring(const char *str, unsigned char *dest, int posx, int posy, int destwidth, int destheight) {
 	int i=0;
 	int pos=posx;
 
